src/main/java/org/broadinstitute/consent/http/models/ApprovedDataset.java

```diff
@@ -1,24 +1,23 @@
 package org.broadinstitute.consent.http.models;
 
-import java.util.Date;
+import java.sql.Timestamp;
 
 public class ApprovedDataset {
 
   private int alias;
   private String darCode;
   private String datasetName;
   private String dacName;
-  private Date approvalDate;
   private String datasetIdentifier;
+  private final Timestamp expirationDate;
 
-  public ApprovedDataset(int alias, String darId, String datasetName, String dacName,
-      Date approvalDate) {
+  public ApprovedDataset(int alias, String darId, String datasetName, String dacName, Timestamp expirationDate) {
     this.alias = alias;
     this.darCode = darId;
     this.datasetName = datasetName;
     this.dacName = dacName;
-    this.approvalDate = approvalDate;
     this.datasetIdentifier = Dataset.parseAliasToIdentifier(alias);
+    this.expirationDate = expirationDate;
```

Comment:
This seems like it will require corresponding updates to [client-side code that references approvalDate](https://github.com/search?q=repo%3ADataBiosphere%2Fduos-ui%20approvaldate&type=code).

Reply:
Good callout, Eric! I've taken care of this as part of https://github.com/DataBiosphere/duos-ui/pull/2951 since I was already modifying related code

Reply:
Apologies- I read this far too quickly and totally missed the part about approvalDate being _replaced_ with expirationDate, rather than being removed entirely 🤦‍♂️ 

Elliot, I can also take care of adding the expiration date field, if we want to coordinate a tiny bit on the sequencing of merges.

Reply:
@MatthewBemis - That would be great if you add it to the front end in your ticket!  Thank you!